pkg/ottl/expression_test.go

```diff
@@ -318,3 +319,107 @@ func Test_newGetter(t *testing.T) {
 		assert.Error(t, err)
 	})
 }
+
+func Test_StringGetter(t *testing.T) {
```

Comment:
I think the typed Getter test suites could probably be combined into a single suite and just provide a `StandardTypeGetter` for each test case. Not a big deal, but it would simplify adding new types a little bit.